tools/upgrader.py

```diff
@@ -0,0 +1,37 @@
+import os
+import argparse
+
+
+def upgrade(frm,to):
+
+    if frm == "2.4.0" and to == "2.5.0":
+        print(f"UPGRADING FROM {frm} TO {to} ...")
+
+        os.system("curl -O https://raw.githubusercontent.com/IBM/ibm-spectrum-scale-csi/v2.5.0/generated/installer/ibm-spectrum-scale-csi-operator.yaml")
+        os.system("kubectl apply -f ibm-spectrum-scale-csi-operator.yaml")
+        os.system("kubectl get pod -n ibm-spectrum-scale-csi-driver")
+        os.system("kubectl describe pod ibm-spectrum-scale-csi-operator-8947b76cb-k8ggx -n ibm-spectrum-scale-csi-driver")
```

Comment:
does this work? I don't think this pod name is static so it probably won't find this pod.  You would need to get the pod name dynamically, for example:

`kubectl describe pod $(kubectl get pod -lapp.kubernetes.io/instance=ibm-spectrum-scale-csi-operator -n ibm-spectrum-scale-csi-driver -ojson | jq -r '.items[0].metadata.name') -n ibm-spectrum-scale-csi-driver`